apps/contributor/contracts/Contributor.sol

```diff
@@ -60,23 +60,23 @@ contract Contributor is AragonApp {
     ContributorAccountUpdated(id, oldAccount, newAccount);
   }
 
-  function updateContributorIpfsHash(uint32 id, bytes32 ipfsHash, uint8 hashFunction, uint8 hashSize) public isInitialized auth(MANAGE_CONTRIBUTORS_ROLE) {
+  function updateContributorIpfsHash(uint32 id, bytes32 hashDigest, uint8 hashFunction, uint8 hashSize) public isInitialized auth(MANAGE_CONTRIBUTORS_ROLE) {
```

Comment:
Why do we need that? If we need it I would rename it to `updateContributorProfile` like the event.

Reply:
very true. 
I think we need it to be able to update the user's details - like profile picture or whatever.

Reply:
Any and all user details actually. Everything except the Ethereum account/address.